en_ewt-ud-train.conllu

```diff
@@ -153704,7 +153704,7 @@
 
 # sent_id = answers-20111108103333AA3eSCk_ans-0019
 # text = $ometime$ it is hard to explain that you are making an INTERNATIONAL call and all of the phone companie$involved what $ome of your money.
-1	$ometime$	sometimes	ADV	RB	_	4	advmod	4:advmod	_
+1	$ometime$	sometimes	ADV	RB	Typo=Yes	4	advmod	4:advmod	CorrectForm=Sometimes
```

Comment:
I think the $ signs are intentional, so not a typo exactly, and not an abbreviation, but a stylized spelling? Not sure how we should represent that.

Reply:
Could you also add the feature for "companie$" and "$ome"?

Reply:
Add please add `CorrectForm=sometimes`